### ISSUE TITLE
Backstopjs can't find the relative path config in backstop.json

### ISSUE THREAD

**Issue by JackZhang1988** [garris/BackstopJS#234]
My file structure like this:
--node_modules
--project
----shop
------backstop.json

and the scripts path in backstop.jso is:

```
"paths": {

        "bitmaps_reference": "../../project/shop/backstop_data/bitmaps_reference", 

        "bitmaps_test": "../../project/shop/backstop_data/bitmaps_test",

        "compare_data": "../../project/shop/backstop_data/bitmaps_test/compare.json",

        "casper_scripts": "../../project/shop/backstop_data/casper_scripts"
    }
```

i run backstop test in node_modules/backstop/ by npm run: 
`npm run test -- --configPath=/root/wdfe_autotest/project/shop/backstop.json;`

And I run this in my mac os, everything is ok.

but when i run this in my Linux server (Red Hat Enterprise Linux Server release 6.5 ), an error occurred and shows that: "../../project/shop/backstop_data/casper_scripts/onReady.js was not found"

i'm sure that there has the file onReady.js.

And i try to debug backstop.js, i output '__dirname' and 'path.backstop' variable and shows that is: 

```
__dirname :  /root/wdfe_autotest/node_modules/.npminstall/backstopjs/1.3.5/backstopjs/gulp/util

paths.backstop:  /root/wdfe_autotest/node_modules/.npminstall/backstopjs/1.3.5/backstopjs
```

I think the problem is there, there is an '.npminstall' directory, so the backstopjs find the wrong relative path.

But i don't understand why there is an '.npminstall' directory. My backstopjs is exactly in node_modules/backstopjs. 

i tried search this in Google, but i got nothing help. Is there anyone know how to solve my problem?

**Comment by garris** [comment]
I agree with you -- the install path is very unusual.   I have never seen this issue.

Maybe you could try installing again in another location -- just for a sanity check.

What version of mom & node are you on?

**Comment by JackZhang1988** [comment]
@garris my node is 4.0.0 and npm is 3.8.8

**Comment by garris** [comment]
Closed by #259
